### ISSUE TITLE
Controlled component moves weird when dragging whole range

### ISSUE THREAD

**Issue by chrisnok** [n3r4zzurr0/react-range-slider-input#8]
Dragging a whole range with a react controlled value causes the slider to move with a different speed than the mouse cursor. It looks fine when you move the slider fast, but when moving it slowly the slider "fly away" from the cursor.

You can see the effect in the sandbox demo in the "Controlled" component by dragging the whole slider slowly or fast. It does not happen for "Default" nor if you only drag one end. However, it does appear like the "Default" might be self correcting itself...

Tested in Chrome and Safari on Mac.

**Comment by alpinisme** [comment]
Yes, I can verify that this exists on Firefox on Mac as well, and on an iPhone.

**Comment by n3r4zzurr0** [comment]
Thanks for reporting the issue! It has been fixed now. Sorry it took way longer than it should have. :(
